MatrixSDK/MXRestClient.m

```diff
@@ -318,6 +321,9 @@ - (MXHTTPOperation*)registerOrLoginWithUser:(MXAuthAction)authAction user:(NSStr
                              // Workaround: HS does not return the right URL. Use the one we used to make the request
```

Comment:
I know this isn't really part of this PR, but is this something we need to fix in synapse, and if so is there a bug for it?